react/components/Checkbox/index.js

```diff
@@ -65,6 +69,7 @@ class Checkbox extends PureComponent {
           </div>
           <input
             checked={checked}
+            ref={this.props.forwardedRef}
```

Comment:
calls the `forwardedRef` of the object on [line 25](https://github.com/vtex/styleguide/pull/1014/files#diff-3b1850cbf3bfc7acf62753d19d2772efR25)

```suggestion
            ref={forwardedRef}
```

Reply:
Done